### PR TITLE
{bio}[goolf/1.4.10] Raster3D v3.0-2

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 7 out of 7 (4 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/18b324d87de33341f9c158e8dd5c9597 for a full test report.

File: easybuild/easyconfigs/l/libjpeg-turbo/libjpeg-turbo-1.3.0-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+name = 'libjpeg-turbo'
+version = '1.3.0'
+
+homepage = 'http://sourceforge.net/libjpeg-turbo/'
+description = """ libjpeg-turbo is a fork of the original IJG libjpeg which 
+ uses SIMD to accelerate baseline JPEG compression and decompression. libjpeg 
+ is a library that implements JPEG image encoding, decoding and transcoding.
+"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
```
Comment (boegel) [inline easybuild/easyconfigs/l/libjpeg-turbo/libjpeg-turbo-1.3.0-goolf-1.4.10.eb]: @pescobar existing easyconfigs for `libjpeg-turbo` have `toolchainopts = {'pic': True}` here, and also include custom `sanity_check_paths`, please do that here too for consistency?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 4 (4 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/39d4ee401691ae3599bd86ae6120ef5e for a full test report.

**Comment by boegel** [comment]
@pescobar missing dep?

```
local.c:61:16: fatal error: gd.h: No such file or directory
 #include "gd.h"
```

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**